presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -300,30 +300,30 @@ public void testDescribeTable()
     @Test
     public void testExplainAnalyze()
     {
-        assertExplainAnalyze("EXPLAIN ANALYZE SELECT * FROM orders");
-        assertExplainAnalyze("EXPLAIN ANALYZE SELECT count(*), clerk FROM orders GROUP BY clerk");
-        assertExplainAnalyze(
+        assertExplainAnalyzeSucceeds("EXPLAIN ANALYZE SELECT * FROM orders");
```

Comment:
What's the difference between these cases and cases in `TestDistributedEngineOnlyQueries#testExplainAnalyze`?

`assertExplainAnalyzeSucceeds` still checks basic `(?s:.*)CPU:.*, Input:.*, Output(?s:.*)`, but why we don't want to check if for ``TestDistributedEngineOnlyQueries#testExplainAnalyze` too?

Reply:
I do not think verifying that output contains `CPU:.*, Input:.*, Output` buys any additional safety actually here, so i optimized this for readability and maintainability.
If you think validating `CPU:.*, Input:.*, Output` is useful, i can think how to accommodate that.

Reply:
> I do not think verifying that output contains CPU:.*, Input:.*, Output buys any additional safety actually here, so i optimized this for readability and maintainability.

Why not to remove it from other tests too then and just leave it in one test?

I'm concerned that execution has non-trivial impact in `EXPLAIN ANALYZE` queries (as we have seen). There might be more non-trivial interactions in the future (e.g regarding some lazy reads, etc). Previously, there was almost no `EXPLAIN` test coverage. IMO we should harden explain assertions (by running them on many different plans including the ones in `TestDistributedEngineOnlyQueries#testExplainAnalyze` method) and not relax it.